### ISSUE TITLE
Ignore check for user

### ISSUE THREAD

**Issue by kceb** [mergeability/mergeable#342]
We use dependabot for automatic creation of dependency upgrade PRs. If their PRs do not pass mergeable's validation, it is considered invalid. Is there a way to add users to an "allow list" in the config?

Currently we are using a hack that utilizes an "or" validation and checks for a "dependencies" label and mention of "dependabot" in the PR description (dependabot's default PR template)

I found this https://github.com/mergeability/mergeable/issues/145
but it uses the similar label hack we are using.

**Comment by shine2lay** [comment]
@kceb is this "allow list" a list of PR authors in which the mergeable will run? 
Would an "ignore list" be better for your case?

**Comment by kceb** [comment]
ah yea I mis-spoke. The list of users would be authors for which mergeable _wont_ run

@shine2lay

**Comment by shine2lay** [comment]
> add them to an "allow list" for them to be able to merge the PR without mergeable checks 

I don't think a github app can control the flow of allowing someone to merge the PR without mergeable checks. The solution i can think of is to create a "pass" check without running validations. I think this will solve edge case for repo that have mergeable as a "required" for merging as well. WDYT?

@kceb

**Comment by shine2lay** [comment]
@kceb we implemented a new feature for `filters` that checks whether the validation should be ran based on the author.

```
  filter:
      - do: author
        must_exclude:
          regex: '^dependabot-preview$'
          regex_flag: none
```

implemented in #505
